it/uk/gov/hmrc/soletraderidentificationfrontend/assets/TestConstants.scala

```diff
@@ -18,16 +18,18 @@ package uk.gov.hmrc.soletraderidentificationfrontend.assets
 
 import play.api.libs.json.{JsArray, JsObject, Json}
 import uk.gov.hmrc.soletraderidentificationfrontend.models.BusinessVerificationStatus._
-import uk.gov.hmrc.soletraderidentificationfrontend.models.SoleTraderDetailsMatching.{DetailsMismatch, SuccessfulMatch}
+import uk.gov.hmrc.soletraderidentificationfrontend.models.SoleTraderDetailsMatching.{DetailsMismatch, KnownFactsNoContentKey, SuccessfulMatch}
 import uk.gov.hmrc.soletraderidentificationfrontend.models._
 
 import java.time.LocalDate
 import java.util.UUID
 
 object TestConstants {
 
+  val seventeen: Int = 17
```

Comment:
Could we change the name of this val to something more meaningful, please? Like testMinAge or something similar?